### ISSUE TITLE
Add 'go to line' functionality to the editor

### ISSUE THREAD

**Issue by rat-moonshine** [prominic/Moonshine-IDE#241]
Can be an option somewhere inside the 'Edit' menu.

**Comment by rat-moonshine** [comment]
Added Ctrl+L or CMD+L to open Go To Line window. Open for test now.

**Comment by piotrzarzycki21** [comment]
@rat-moonshine The only change which I would make here is replace `prompt="#"` with something more meaningful. Maybe some text. Enter number... 
Apart of that it is very useful feature! :)

**Comment by rat-moonshine** [comment]
We already have input label and window title saying this. So again in prompt could be redundant message;  thus I choose to mark the prompt as some digit/number.

**Comment by JoelProminic** [comment]
This looks fine.  I checked:
- This works quickly for a very large file (100,000+ lines)
- It is not possible to enter a negative number
- "0" will go to line 1
- If you enter a value outside of the line range, the prompt will not do anything.  The next valid line number will be fine
- The line numbers are consistent with the numbers returned by stacktraces, so this will be useful for debugging.

**Comment by JoelProminic** [comment]
Retesting this feature on MacOS today, I noticed some problems that I didn't see during my previous tests:
1. Open a file
2. CMD-L  (at one point in my tests, CMD-L did not work but CTRL-L did.  I was not able to reproduce this)
3. Enter a valid line number and submit
4. BUG: the updated editor tab does not have focus
5. Click on the tab header.  The cursor was placed at the correct line.

I did not see this error on Windows.

**Comment by rat-moonshine** [comment]
When I tested this on Windows, I able to reproduce this in Windows too, in certain steps:

1. Make sure you have a class file opened to editor
2. Restart Moonshine
3. Moonshine restarted and auto-opened the last opened files in editor
4. I can see the cursor is blinking at 0th index of first line in opened editor
5. **I received a version update alert of Moonshine as popup**
6. Closed the alert by clicking on its button
7. The opened editor at background (of the popup) has lost cursor focus and there was no blinking cursor
8. At this moment if I press CTRL+L it does open the jump to line popup but cursor do not visible until I clicked on the editor's tab

This seems something newly introduced. I need to check.
